src/main/resources/db/migration/table_initialization.sql

```diff
@@ -0,0 +1,69 @@
+create schema "advisory";
+set schema 'advisory';
+
+create type phone_numbers_type as enum ('home', 'work', 'mobile');
+create type advisor_role as enum ('associate', 'partner', 'senior');
+create type application_status as enum ('new', 'assigned', 'on_hold', 'approved', 'canceled', 'declined');
+create type user_type as enum ('advisor', 'applicant');
+
+
+create table if not exists users
+(
+    id        bigserial,
+    email     text not null,
+    user_name text not null,
+    constraint users_pk primary key (id),
+    constraint email_uq unique (email),
+    constraint user_name_uq unique (user_name)
+);
+
+create table if not exists advisors
```

Comment:
Hi! According to task: "Any person who works with the system (advisor, applicant) is a user". How does your schema ensure this?